packages/builder/src/stores/builder/permissions.js

```diff
@@ -1,11 +1,11 @@
-import { writable } from "svelte/store"
+import { writable, get } from "svelte/store"
 import { API } from "api"
 
 export function createPermissionStore() {
-  const { subscribe } = writable([])
+  const store = writable({})
 
   return {
-    subscribe,
+    subscribe: store.subscribe,
```

Comment:
You could refactor the store to extend from `BudiStore` and it would have boilerplate like this sorted 👍

Reply:
Ended up removing the changes to this store, so I'll punt on this for now 🙏